runtime/port/CMakeLists.txt

```diff
@@ -195,6 +195,8 @@ omr_add_exports(j9prt
 	j9port_startup_library
 )
 
+target_enable_ddr(j9prt j9ddr EARLY_SOURCE_EVAL)
```

Comment:
`EARLY_SOURCE_EVAL` why is that added here?  A comment in the file explaining it would be good